### PR TITLE
fix: nil pointer deref

### PR DESCRIPTION
**Pull request by LiquidTheDangerous**

### CODE REVIEW

File: proxy/proxy.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (pxy *Proxy) Start() {
 		go func() {
 			pkt, err := packet.ReadHttpPacket(conn)
 			if err != nil {
-				log.Debug("[PROXY] Error while parsing request: ", string(pkt.Raw()))
```
Comment (LiquidTheDangerous) [inline proxy/proxy.go:66]: Sadly, didn't notice this in #138

**Review by xvzc — APPROVED** [review]

**Comment by xvzc** [comment]
We should also find what causes the err

**Comment by LiquidTheDangerous** [comment]
Perhaps the connection was closed, EOF was returning

**Comment by xvzc** [comment]
Yeah hopefully

**Pull request merged**